app/models/featured_tag.rb

```diff
@@ -58,5 +60,6 @@ def validate_featured_tags_limit
   def validate_tag_name
     errors.add(:name, :blank) if @name.blank?
     errors.add(:name, :invalid) unless @name.match?(/\A(#{Tag::HASHTAG_NAME_RE})\z/i)
+    errors.add(:name, I18n.t('featured_tags.errors.duplicate_hashtag')) if FeaturedTag.by_name(@name).where(account_id: account_id).exists?
```

Comment:
Maybe this should trigger the normal `:taken` error message?

Reply:
Fix it.